dashboard/dashboard-user.md

```diff
@@ -0,0 +1,64 @@
+---
+title: TiDB Dashboard 用户管理
+summary: 了解如何创建 TiDB Dashboard 自定义 SQL 用户
+---
+
+# TiDB Dashboard 用户管理
+
+从 TiDB 5.3 开始，TiDB Dashboard 支持使用自定义的 SQL 用户登录访问，你可以为自定义 SQL 用户指定所需要的权限。
+
+- 当所连接的 TiDB 服务器未启用[安全增强模式 (SEM)](/system-variables.md#tidb_enable_enhanced_security) 时，要访问 TiDB Dashboard，需要为自定义的 SQL 用户赋予 `ALL PRIVILEGES` 权限或者以下**所有**权限：
+
+    - PROCESS
+    - SHOW DATABASES
+    - CONFIG
+    - DASHBOARD_CLIENT 或者 SUPER
+
+- 当所连接的 TiDB 服务器启用了[安全增强模式 (SEM)](/system-variables.md#tidb_enable_enhanced_security) 时，要访问 TiDB Dashboard，需要为自定义的 SQL 用户赋予 `ALL PRIVILEGES` 权限或者以下**所有**权限：
+
+    - PROCESS
+    - SHOW DATABASES
+    - CONFIG
+    - DASHBOARD_CLIENT 或者 SUPER
+    - RESTRICTED_TABLES_ADMIN
+    - RESTRICTED_STATUS_ADMIN
+    - RESTRICTED_VARIABLES_ADMIN
+
+- 当需要自定义的 SQL 用户在登录 TiDB Dashboard 后能修改界面上的各项配置时，请为自定义的 SQL 用户赋予以下**任意**权限时。否则 SQL 用户只有读权限，无法修改配置。
+
+    - ALL PRIVILEGES
+    - SUPER
+    - SYSTEM_VARIABLES_ADMIN
+
+如果自定义的 SQL 用户的权限未满足要求时，该用户将无法登录 TiDB Dashboard。如下图所示：
+
+![insufficient-privileges](/media/dashboard/dashboard-user-insufficient-privileges.png)
```

Comment:
I can't see the picture at the rich diff.
Is it still work?

Reply:
![image](https://user-images.githubusercontent.com/1284531/140246877-efc30ca0-e59b-493a-9032-4b2a6072d626.png)